### DISCUSSION TITLE
Give feedback to new gopackages checker API

### DISCUSSION THREAD

**Discussion by nightlyone** [golangci/golangci-lint#2919] [General]
Hi @ldez,

Just a heads-up that you have a great opportunity to give feedback to a new proposed checker API at https://go-review.googlesource.com/c/tools/+/411907

Since you maintain the largest Go linter integration project you are in a unique position to share your perspective here.

Maybe Alan Donovan already reached out to you?

**Comment by ldez** [comment]
Hi @nightlyone,

Thank you for pinging me on this topic, I try to provide feedback.

**Reply by Antonboom** [reply]
Thread for comments – https://github.com/golang/go/issues/61324
